frontend/src/scenes/notebooks/Notebook/migrations/migrate.test.ts

```diff
@@ -13,7 +13,7 @@ describe('migrate()', () => {
                 {
                     type: 'ph-query',
                     attrs: {
-                        query: '{"kind":"InsightVizNode","source":{"kind":"TrendsQuery","properties":{"type":"AND","values":[{"type":"AND","values":[]}]},"filterTestAccounts":true,"dateRange":{"date_to":null,"date_from":"-90d"},"series":[{"kind":"EventsNode","event":"$pageview","name":"$pageview","properties":[{"key":"$referring_domain","type":"event","value":"google|duckduckgo|brave|bing","operator":"regex"},{"key":"utm_source","type":"event","value":"is_not_set","operator":"is_not_set"},{"key":"$host","type":"event","value":["posthog.com"],"operator":"exact"}],"math":"dau"}],"interval":"week","breakdown":{"breakdown_type":"event","breakdown":"$referring_domain"},"trendsFilter":{"compare":false,"display":"ActionsBar"}}}',
+                        query: '{"kind":"InsightVizNode","source":{"kind":"TrendsQuery","properties":{"type":"AND","values":[{"type":"AND","values":[]}]},"filterTestAccounts":true,"dateRange":{"date_to":null,"date_from":"-90d"},"series":[{"kind":"EventsNode","event":"$pageview","name":"$pageview","properties":[{"key":"$referring_domain","type":"event","value":"google|duckduckgo|brave|bing","operator":"regex"},{"key":"utm_source","type":"event","value":"is_not_set","operator":"is_not_set"},{"key":"$host","type":"event","value":["posthog.com"],"operator":"exact"}],"math":"dau"}],"interval":"week","breakdown":{"breakdown_type":"event","breakdown":"$referring_domain"},"compareFilter":{"compare":true,"compare_to":"-4w"},"trendsFilter":{"display":"ActionsBar"}}}',
```

Comment:
Didn't get around to a full review yet, so just a fly-by that I think we need to actually adapt the `migrate.ts` function for notebook nodes with queries to still work.

Reply:
Thanks for the heads up. I wasn't familiar with notebook functionality.

I added the migration code here:
https://github.com/PostHog/posthog/pull/22397/files#diff-fef986adb6581e65a4f5a219af0c0aba9896c6e22ab650ee3912843ff761ff4a